torchsde/_brownian/brownian_interval.py

```diff
@@ -106,80 +101,67 @@ def increment_and_levy_area(self):
         return W, H, A
 
     def _increment_and_space_time_levy_area(self):
-        return self._top.increment_and_space_time_levy_area_cache(self)
-
-    def increment_and_space_time_levy_area_uncached(self):
         # TODO: switch this over to a trampoline?
```

Comment:
If the plan is to not go for the trampoline approach, should this TODO be removed?

Reply:
I'm leaving it there for now as I think adding a trampoline may still be useful later on, if any use case gives even deeper call stacks.